tools/installer/common/installer.include

```diff
@@ -142,9 +142,9 @@ stage_install_common() {
   fi
 
   # l10n paks
-  cp -a "${BUILDDIR}/locales" "${STAGEDIR}/${INSTALLDIR}/"
-  find "${STAGEDIR}/${INSTALLDIR}/locales" -type f -exec chmod 644 '{}' \;
-  find "${STAGEDIR}/${INSTALLDIR}/locales" -type d -exec chmod 755 '{}' \;
+  cp -a "${BUILDDIR}/locales/xwalk" "${STAGEDIR}/${INSTALLDIR}/"
```

Comment:
Yes, mistake here. I think we don't need to change this file.  Its original logic works fine with "/locales/xwalk".